pkg/ddevapp/traefik.go

```diff
@@ -409,6 +409,78 @@ func configureTraefikForApp(app *DdevApp) error {
 		}
 	}
 
+	/* The following section reads the project/.ddev/traefik/dynamic_config.*.yaml files, fills any template placeholders in them with the
```

Comment:
It templates these still? Or not?

Reply:
it's OK with me to template-process those dynamic_config.*.yaml files, but please don't show it in the example.